Because of -Wl,--export-dynamic in app-misc/openrgb, this resources.qrc
conflicts with the openrgb's one. So rename it.

--- a/OpenRGBEffectsPlugin.pro
+++ b/OpenRGBEffectsPlugin.pro
@@ -472,4 +472,4 @@ macx: {
 }
 
 RESOURCES += \
-    resources.qrc
+    resources_effects_plugin.qrc
